### DISCUSSION TITLE
Location not updating on Reservation Movement

### DISCUSSION THREAD

**Discussion by kernny** [sheltermanager/asm3#1527] [Q&A / Help]
When we add a Movement to Reserve an animal pending adoption, the LOCATION of the animal used to update and show that animal as Reserved --> Person reserved to but now it seems that is not changing until the Adoption Movement is finalised. 

See photos - 
1. New animal added into the generic Foster Care location. (We use Foster Care instead of Shelter as we only have carers.)

![Screenshot 1](https://github.com/user-attachments/assets/e40e86c4-7e20-4a79-93cb-417b90e5dec7)

2. Add Movement to assign Foster Carer - Location status changes to show FOSTER - Foster carer name.

![Screenshot 2](https://github.com/user-attachments/assets/20a4d160-38e3-415a-b31b-5a1d44587ab8)

3. Add Movement to RESERVATION to reserve the animal for adoption. This is when the location used to change to show the Reserved --> Persons name. You can see in the photo, the location does not change. It only update once the Reservation is changed to Adoption.

![Screenshot 3](https://github.com/user-attachments/assets/bf3ef536-ea7f-4079-812d-61b015dc6660)
 

Is there a different way we should be adding the animal or making the movement? How do we get the Location status to change back to how it used to work?

**Comment by bobintetley** [comment]
I don't think Location ever showed the reservation info? Assuming that the reserve is the only thing keeping an animal from being adoptable, the reserve info is shown in the adoption status banner on the right:

![image](https://github.com/user-attachments/assets/e39e5dc9-1182-4402-a40f-1df6872795d2)
